docs/src/snap/explanation/about.md

```diff
@@ -34,6 +34,8 @@ be: run it in several local containers or VMs for example, or use it on
 public/private cloud instances. 
 For deploying with [Juju], a machine [charm] to deploy
 the snap is also available.
+For deployment using Cluster API, additional resources have also been
```

Comment:
For deployment using Cluster API, a Bootstrap and Control-plane provider have also been created.